libafl_frida/src/asan_rt.rs

```diff
@@ -55,14 +60,43 @@ const ANONYMOUS_FLAG: MapFlags = MapFlags::MAP_ANON;
 #[cfg(not(target_vendor = "apple"))]
 const ANONYMOUS_FLAG: MapFlags = MapFlags::MAP_ANONYMOUS;
 
+// sixteen general purpose registers are put in this order, rax, rbx, rcx, rdx, rbp, rsp, rsi, rdi, r8-r15, plus instrumented rip, accessed memory addr and true rip
+#[cfg(target_arch = "x86_64")]
+pub const ASAN_SAVE_REGISTER_COUNT: usize = 19;
+
+pub const ASAN_SAVE_REGISTER_NAME: [&str; ASAN_SAVE_REGISTER_COUNT] = [
+    "Rax",
```

Comment:
Should these not have a lowercase first char?

Reply:
ok